platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -1,84 +1,198 @@
+
+1. Edit the `proxy.yml` file and set the following variables:
+
+    - `CONNECT_REDIS_ADDRESS`: Specify the host name or IP address of the Redis server that you configure for Platform.
+    - `CONNECT_PROXY_URL`: Specify a base domain name for the connect proxy subdomain. For example, if you specify `example.com` then `connect.example.com` is the fully qualified domain name for the proxy.
+    - `PLATFORM_URL`: Specifies the base URL for your Platform installation, such as `https://tower.example.com/`.
+
+1. Generate an RSA public/private key pair. A key size of at least 2048 bits is recommended. In the following example, the `openssl` command is used to generate the key pair:
+
+    ```shell
+      openssl genrsa -out private.pem 2048
+      openssl rsa -pubout -in private.pem -out public.pem
+    ```
+
+1. Apply a base64 encoding to the PEM file that you created in the previous step:
+
+    ```
+    base64_pem=$(cat public.pem | base64)
+    ```
+
+1. Create a Secret file named `secret.yml`, and set the `oidc.pem` key to the value of the base64-encoded public/private key pair:
+
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: platform-oidc-certs
+      namespace: platform-stage
+    data:
+      oidc.pem: <BASE64_ENCODED_PEM_FILE>
+    ```
+
+1. Create the Secret:
+
+    ```
+    kubectl apply -f secret.yml
+    ```
+
+1. Edit the `tower-cron.yml` file and uncomment the `volumes.cert-volume`, `volumeMounts.cert-volume`, and `env.TOWER_OIDC_PEM_PATH` fields so that the public/private key pair is available to Platform.
+
+1. Apply the configuration change to Platform:
+
+    ```
+    kubectl apply -f tower-cron.yml
+    ```
+
+1. Edit the ConfigMap named `platform-backend-cfg` in the `configmap.yml` for Platform.
+    -  Add the following environment variables:
+
+       - `TOWER_DATA_STUDIO_CONNECT_URL`: Specifies the URL of the Data Studios connect proxy, such as `https://connect.tower.example.com/`.
+       - `TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN`: Use the same base64-encoded access token that you specified in the Secret named `connect-proxy-secret`.
+
+    - If you want to restrict Data Studios to specific organization workspaces, list each workspace ID in the `TOWER_DATA_STUDIO_ALLOWED_WORKSPACES`.
+    - If you want to allow access to Data Studios in every organizational workspace, do not include the `TOWER_DATA_STUDIO_ALLOWED_WORKSPACES`.
+
+1. Optional: If you want to allow access to Data Studios in every organizational workspace, edit the ConfigMap named `tower-yml` in the `configmap.yml` and include the following stanza with your existing configuration:
+
+    ```yaml
+    data:
+      tower.yml: |-
+        tower:
+          data-studio:
+            allowed-workspaces: null
+    ```
+
+1. Apply the updated configuration: `kubectl apply -f configmap.yml`
```

Comment:
For consistency with other instructions, please split the command to a new line (GH editor is not understanding this in suggestion mode)